pkg/toolsets/core/toolset.go

```diff
@@ -30,8 +30,33 @@ func (t *Toolset) GetTools(o api.Openshift) []api.ServerTool {
 }
 
 func (t *Toolset) GetPrompts() []api.ServerPrompt {
-	// Core toolset prompts will be added in Feature 3
-	return nil
+	return []api.ServerPrompt{
```

Comment:
To keep things concise in this file, it'd be better if we followed the same pattern used for tools: define a `initHealthChecks()` function within the `health_check.go` file and reuse it here.